topics/statistics/tutorials/deeplearning_intro/tutorial.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
There is no text for deep learning I could find? I could find only images, data.yml files. I will create a fresh one.